### PR TITLE
add required files and functions from chanjo-report to make translati…

### PR DESCRIPTION
**Pull request by robinandeer**
…ons work :stuck_out_tongue:

### CODE REVIEW

**Comment by moonso** [comment]
Is there anything more for me to do than reviewing the pull request?

**Comment by robinandeer** [comment]
Not really, I've tested it locally and it works

File: scout/settings.py [hunk 1]
```diff
@@ -31,6 +31,9 @@ class BaseConfig(object):
   MAIL_PASSWORD = 'yourpass'
   MAIL_DEFAULT_SENDER = MAIL_USERNAME
 
+  REPORT_LANGUAGE = 'en'
+  ACCEPT_LANGUAGES = ['en', 'sv']
```
Comment (moonso) [inline scout/settings.py]: Nice! It is possible to choose language on the fly?

**Comment by moonso** [comment]
:kimono:

File: scout/settings.py (hunk 1 above)
Comment (robinandeer) [inline scout/settings.py]: yep as query arg: `?lang=sv`

**Pull request merged**